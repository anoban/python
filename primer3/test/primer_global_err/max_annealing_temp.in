SEQUENCE_ID=max_annealing_temp
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_ANNEALING_TEMP=120.0
=
