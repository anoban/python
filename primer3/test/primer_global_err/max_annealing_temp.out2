../src/primer3_core: Annealing temperature higher than 100 C
