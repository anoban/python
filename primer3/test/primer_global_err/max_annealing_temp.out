SEQUENCE_ID=max_annealing_temp
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_ANNEALING_TEMP=120.0
PRIMER_ERROR=Annealing temperature higher than 100 C
=
